updateCertificationStatusApp.sh

```diff
@@ -5,4 +5,20 @@
 # 15 5 * * * cd /tempApps && ./updateCertificationStatusApp.sh
 # This will run it at 0515 UTC, which (depending on DST) is 0015 EST
 
-java -jar certStatusApp-jar-with-dependencies.jar
\ No newline at end of file
+# create timestamp and filename
+TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+log=logs/log.certStatusApp.$TIMESTAMP.txt
+
+# deal with spaces in filenames by saving off the default file separator (including spaces)
+# and using a different one for this application
```

Comment:
"Just to be safe" is cause of so much code debt and cargo cult programming. If you _know_ why it's there, and choose to use it, that's fine, but if you don't, or if you're not sure, then it just means it's left there to confuse someone later on